.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: build.yml
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '0 8 1 * *'
```

Comment:
Was there a regular scheduled build for this already that needed to be replaced? Or was this just a copy-and-paste from the hapi-gapi repo?

Reply:
This was copy and pasted! I'll remove the schedule part as it's not needed.